### PR TITLE
WIP: Update hu.json

### PR DESCRIPTION
**Pull request by kistom4**

### CODE REVIEW

File: locales/hu.json [hunk 1]
```diff
@@ -1111,11 +1111,11 @@
...
       "keywords": {
-        "gb": "gb",
-        "great_building": "great building",
+        "gb": "nevi",
+        "great_building": "nevezetes épület",
         "fp": "fp",
-        "forge_point": "forge point",
-        "medals": "medals",
+        "forge_point": "forge pont",
+        "medals": "medál,
```
Comment (Heziode) [inline locales/hu.json]: Missing double quote at the end (after **medál** and before comma)

Comment (kistom4) [reply locales/hu.json]: Corrected it.

**Comment by Heziode** [comment]
@kistom4, this PR is in work in progress or it is complete ?

**Comment by kistom4** [comment]
> @kistom4, this PR is in work in progress or it is complete ?

In progress.

**Comment by Heziode** [comment]
Ok, I have updated the name of this PR to prefix by « WIP: ».
Remove this prefix when this PR will be completed.

**Comment by Heziode** [comment]
I have rename the `dev` branch to `develop`. Can you re-open the pull ?

**Pull request closed without merging**